avocado/utils/network/interfaces.py

```diff
@@ -571,3 +571,29 @@ def restore_slave_cfg_file(self):
                     os.remove(slave_config)
             except Exception as ex:
                 raise NWException(f"Could not restore the config file {ex}")
+
+    def are_packets_lost(self, peer_ip, count, option=None, flood=False):
```

Comment:
s/option/options/g

Reply:
Adding arguments as count and options only. 
Count we can have separate i feel as it will be very easy of any user to add count for number of packets
And option will be for any option user wants to add to ping command. including flood(-f)